### ISSUE TITLE
Preview button should be with accessibility attributes

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#8088]
Select a site 

![image](https://github.com/user-attachments/assets/11a84d72-87ae-4dae-bc3e-73338ea29825)

**Issues**: 
1. Add tooltip for new `Preview` button

2. Add accessibility attributes - Buttons must have discernible text:
  https://dequeuniversity.com/rules/axe/4.8/button-name

```
Ensure that each button element and elements with role="button" have one of the following characteristics:

Inner text that is discernible to screen reader users.
Non-empty aria-label attribute.
aria-labelledby pointing to element with text which is discernible to screen reader users.
role="presentation" or role="none" (ARIA 1.1) and is not in tab order (tabindex="-1").
```
